decidim-core/app/serializers/decidim/importers/participatory_space_components_importer.rb

```diff
@@ -62,6 +62,15 @@ def import_component_specific_data(component, serialized, user)
         specific_importer = component.manifest.specific_data_importer_class.new(component)
         specific_importer.import(serialized[:specific_data], user)
       end
+
+      def override_step_settings_ids(attributes, step_settings)
+        return if step_settings.nil?
+
+        @participatory_space.steps.each do |step|
```

Comment:
can you link the place where the @participatory_space.steps are being setted from the imported json?

Reply:
https://github.com/CodiTramuntana/decidim/blob/341ccbf11e04eab2c87eb58b4208b66046d15111/decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/import_participatory_process.rb#L42